vmdb/app/models/ems_kubernetes.rb

```diff
@@ -11,19 +11,40 @@ def self.description
     @description ||= "Kubernetes".freeze
   end
 
-  def self.raw_connect(hostname, port, api_version)
+  def self.raw_connect(uri, api_version)
```

Comment:
I think we wanna see this stay with the `hostname` and `port` parameters instead of the `uri` directly.  Then, maybe add a class method that builds the URI given the host and port parameters.

That way callers of this exposed class method don't need to know that the correct URI value includes the `"/api"` piece.